app/views/home/feedback/feedback.html.haml

```diff
@@ -14,45 +14,45 @@
     .feedback-form
       .feedback-title-container
         %h2.feedback-title
-          Leave your feedback
+          = t("home.feedback.leave_your_feedback")
         %hr#feedback-title-line/
       - unless session[:user].nil?
         %p.feedback-text
-          Hi
+          = t("home.feedback.feedback_hi_text")
           %font 
             = session[:user].username
             ,
       %p.feedback-email
-        you can use the form below or email us directly at:
+        = t("home.feedback.feedback_info_text")
         %a{:href => "mailto: #{$SUPPORT_EMAIL}"}= $SUPPORT_EMAIL
       - if session[:user].nil?
         %p.feedback-input-title
-          Email
+          = t("home.feedback.email")
           %font{:color => "red"} *
         %input.feedback-input{:name => "email", :type => "email"}/
         %p.feedback-input-title
-          Name
-          %font.register-optional (optional)
+          = t("home.feedback.name")
+          %font.register-optional= t("home.feedback.optional")
```

Comment:
For '(optional)' it's better to keep the parentheses in the UI, so we keep the local lang files clean with only texts,
Like this for example:
`"(#{t('home.feedback.optional')})"`